src/main/java/in/twizmwaz/cardinal/module/modules/damageIndicator/PlayerBoundingBox.java

```diff
@@ -0,0 +1,183 @@
+package in.twizmwaz.cardinal.module.modules.damageIndicator;
+
+import com.google.common.base.Optional;
+import in.twizmwaz.cardinal.Cardinal;
+import in.twizmwaz.cardinal.module.modules.team.TeamModule;
+import in.twizmwaz.cardinal.util.PacketUtils;
+import in.twizmwaz.cardinal.util.Teams;
+import net.minecraft.server.DataWatcher;
+import net.minecraft.server.DataWatcherRegistry;
+import net.minecraft.server.Packet;
+import net.minecraft.server.PacketPlayOutEntity;
+import net.minecraft.server.PacketPlayOutEntityDestroy;
+import net.minecraft.server.PacketPlayOutEntityTeleport;
+import net.minecraft.server.PacketPlayOutSpawnEntityLiving;
+import org.apache.commons.lang.ArrayUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.craftbukkit.entity.CraftPlayer;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import org.bukkit.event.player.PlayerMoveEvent;
+import org.bukkit.event.player.PlayerTeleportEvent;
+import org.bukkit.event.player.PlayerUseUnknownEntityEvent;
+import org.bukkit.util.Vector;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class PlayerBoundingBox implements Listener {
```

Comment:
@ElliottWhyman yeah, i know what it does, i just didn't understead twiz's question